invenio_rdm_records/services/communities/service.py

```diff
@@ -185,6 +186,21 @@ def remove(self, identity, id_, data, uow):
         )
         communities = valid_data["communities"]
         processed = []
+
+        if current_app.config.get("RDM_ENSURE_RECORD_COMMUNITY_EXISTS"):
+            rec_communities = current_rdm_records.records_service.read(
+                identity, id_
+            ).to_dict()["parent"]["communities"]
+            if len(rec_communities["ids"]) <= 1:
+                errors.append(
+                    {
+                        "community": f'{rec_communities["ids"][0]}',
+                        "message": _(
+                            "Permission denied, record should have at least one community!"
+                        ),
+                    }
+                )
+                return [{"community": f'{rec_communities["ids"][0]}'}], errors
```

Comment:
If I don't return the community, I will get console log errors, Any ideas on how to notify the user with a helpful message would be greatly appreciated.